notify/notify.yaml

```
@@ -462,7 +464,9 @@ components:
             cursors:
               type: object
               properties:
-                $ref: '#/components/schemas/after'
+                after:
```

ok I think we actually both a little wrong. I think the correct thing to do is:

```suggestion
                after: 
                    $ref: '#/components/schemas/after
```

the reference schema is actually correct.